src/Pages/ProfilePage.tsx

```diff
@@ -31,14 +56,56 @@ function ProfilePage() {
-            alt="profile picture"
-          />
+          {userImg ? (
+            <img
+              style={{ width: "250px", height: "250px", borderRadius: "50%" }}
+              src={userImg}
+              alt="profile picture"
+            />
+          ) : (
+            <img
+              style={{ width: "250px", height: "250px", borderRadius: "50%" }}
+              src="https://thumbs.dreamstime.com/b/man-profile-cartoon-smiling-round-icon-vector-illustration-graphic-design-135443422.jpg"
+              alt="profile picture"
+            />
+          )}
+          <button
```

Comment:
I think this should be more clear. Maybe the button should have 'set display name' text or something?

Reply:
<img width="248" alt="Skärmavbild 2022-12-01 kl  09 55 22" src="https://user-images.githubusercontent.com/90900256/205009142-a3fd5648-56a9-439f-81a1-4f8311c29502.png">
 Is this not clear enough?